docs/platform/how-to/deploy-public-page.md

```diff
@@ -0,0 +1,49 @@
+# Public services
```

Comment:
Should we make the title more active, for example, "Deploying public services" or perhaps "How to deploy a public service"?